src/llmcompressor/transformers/sparsification/sparse_model.py

```diff
@@ -62,6 +62,13 @@ def from_pretrained(
             pretrained_model_name_or_path directory and applied if found
         :return the created model for causal language modeling
         """
+        logger.info(
+            (
+                "`wrap_hf_model_class` is now deprecated. ",
```

Comment:
Can you also add an updated lifecycle? For save/load. See compressed-tensors lifecycle examples:
https://github.com/neuralmagic/compressed-tensors/blob/506cd36829f6fd386a0a2c75ed8a51cc6dcab619/src/compressed_tensors/compressors/base.py#L30